types/lodash/index.d.ts

```diff
@@ -10183,7 +10183,7 @@ declare namespace _ {
          * @param value The value to check.
          * @returns Returns true if value is correctly classified, else false.
          */
-        isWeakSet<T>(value?: any): value is WeakSet<T>;
+        isWeakSet<T extends object>(value?: any): value is WeakSet<T>;
```

Comment:
Just like `isWeakMap`, this method should be made non-generic. Otherwise it'd be a disguised type assertion (see [Common Mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes)).

Reply:
It is needed for compatibility with a fix of TypeScript. See Microsoft/TypeScript#19756. It is blocked by lodash.